clientconn_parsed_target_test.go

```diff
@@ -33,29 +33,75 @@ import (
 	"google.golang.org/grpc/resolver"
 )
 
+func generateTarget(scheme string, target string) resolver.Target {
+	return resolver.Target{URL: *testutils.MustParseURL(fmt.Sprintf("%s:///%s", scheme, target))}
+}
+
 func (s) TestParsedTarget_Success_WithoutCustomDialer(t *testing.T) {
-	defScheme := resolver.GetDefaultScheme()
+	dialScheme := resolver.GetDefaultScheme()
```

Comment:
I believe this should probably try to re-create the initial state in case another test calls `SetDefaultScheme`.   I.e. do `resolver.SetDefaultScheme("passthrough")` and `internal.UserSetDefaultScheme = false`?